pkgs/by-name/op/opengamepadui/package.nix

```
@@ -0,0 +1,154 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  cargo,
+  fetchFromGitHub,
+  alsa-lib,
+  autoPatchelfHook,
+  dbus,
+  fontconfig,
+  gamescope,
+  gnumake,
+  godot_4,
+  godot_4-export-templates,
+  libGL,
+  libX11,
+  libXcursor,
+  libXext,
+  libXfixes,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXrender,
+  libxkbcommon,
+  openssl,
+  pkg-config,
+  vulkan-loader,
+  xorg,
+  udev,
+  libpulseaudio,
+  hwdata,
+  glxinfo,
+  upower,
+  withDebug ? false,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "opengamepadui";
+  version = "0.35.1";
+
+  buildType = "${if withDebug then "debug" else "release"}";
+
+  src = fetchFromGitHub {
+    owner = "ShadowBlip";
+    repo = "OpenGamepadUI";
+    tag = "v${version}";
+    hash = "sha256-741itXn39MxVMTNN9Azx2n5L1PRo5UsRsm6JSeY9nSI=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit src;
+    sourceRoot = "source/${cargoRoot}";
+    hash = "sha256-sTzMewIfKHbmVhSPZgUIzFFz1ahK+PMoQ5oB4GEt8nY=";
+  };
+  cargoRoot = "extensions";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    cargo
+    dbus.dev
+    gnumake
+    godot_4
+    godot_4-export-templates
+    openssl
+    pkg-config
+    rustPlatform.cargoSetupHook
+    xorg.libX11.dev
+    xorg.libXi.dev
+    xorg.libXres.dev
+    xorg.libXtst
+  ];
+
+  buildInputs = [
+    dbus
+    dbus.lib
+    fontconfig
+    fontconfig.lib
+    stdenv.cc.cc.lib
```

The out and lib (+ dev) outputs will be squashed when using `buildInputs`
```suggestion
    dbus
    fontconfig
```

---

Thanks! Still pretty new to Nix packaging. I'll make this change.